docker_test.go

```diff
@@ -57,6 +59,31 @@ func initContainerTest(t *testing.T) []container.ContainerManagerType {
 	return containerManagers
 }
 
+func initNativeDockerTest(t *testing.T, minRtVer, minXrayVer string) func() {
+	if !*tests.TestDocker {
+		t.Skip("Skipping native docker test. To run docker test add the '-test.docker=true' option.")
+	}
+	if minRtVer != "" {
+		servicesManager, err := utils.CreateServiceManager(serverDetails, -1, 0, false)
+		require.NoError(t, err)
+		rtVersion, err := servicesManager.GetVersion()
+		require.NoError(t, err)
+		if !version.NewVersion(rtVersion).AtLeast(minRtVer) {
+			t.Skip("Skipping native docker test. Artifactory version " + corecontainercmds.RtMinVersion + " or higher is required (actual is'" + rtVersion + "').")
+		}
+	}
+	if minXrayVer != "" {
+		initXrayCli()
+		validateXrayVersion(t, minXrayVer)
+	}
+	// Create server config to use with the command.
+	createJfrogHomeConfig(t, true)
+	return func() {
+		oldHomeDir := os.Getenv(coreutils.HomeDir)
```

Comment:
This line should be taken out of this cleanup function, so that the ```oldHomeDir``` indeed stores the old value, before it is changed.